python/fio/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM quay.io/centos/centos:stream8
+
+RUN dnf -y module install python39 && dnf -y install python39 python39-pip
+RUN dnf -y install fio-3.19-3.el8
+RUN mkdir /plugin
+COPY requirements.txt /plugin
+RUN pip3 install --requirement /plugin/requirements.txt
+COPY fio_plugin.py /plugin
+COPY fio_schema.py /plugin
+COPY test_fio_plugin.py /plugin
+COPY mocks /plugin/mocks
```

Comment:
You have both `fixtures` and `mocks` subdirs in the repo with separate references in the README and the code.